docs/configuration/legend.md

```diff
@@ -73,6 +75,16 @@ Items passed to the legend `onClick` function are the ones returned from `labels
 
     // Point style of the legend box (only used if usePointStyle is true)
     pointStyle: String
+
+    // Symbol used on the legend item. if usePointStyles is true, pointStyle is used. Otherwise, if present, the value on dataset is selected.
+    // If none of them are there, the value from legend/labels is selected (default value is rect).
+    legendSymbol: String
+
+	// define if boxWidth is used as width (true) or fontSize (false)
+	legendSymbolLarge: Boolean
+
+    // Width of legend symbol used if legendSymbolLarge is true
+    boxWidth: Number
```

Comment:
My only comment here is that I think it would be better to remove `legendSymbolLarge` and imply it if boxWidth is set. So `if boxWidth !== undefined` then it will be used, otherwise it will be set to fontSize

Reply:
if I'm not wrong, boxWidth is defaulted to 40 so it will be always defined. If we remove default value it may impact current users upgrading to that new version.

Reply:
ahh, I missed that. To make that clearer I would suggest a rename to `useFontSize` or something. Let's wait to rename it until @simonbrunel gets a chance to review this